dotcom-rendering/src/web/components/YoutubeBlockComponent.importable.tsx

```diff
@@ -94,11 +94,11 @@ export const YoutubeBlockComponent = ({
 	const [consentState, setConsentState] = useState<ConsentState | undefined>(
 		undefined,
 	);
-
-	const abTests = useAB();
+	const ab = useAB();
```

Comment:
this is a workaround for an error typescript was throwing in chromatic - it seems a legit error despite not being thrown previously, is storybook / chromatic using a stricter tsconfig in the new setup?